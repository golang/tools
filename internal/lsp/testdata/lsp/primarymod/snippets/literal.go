package snippets

import (
	"golang.org/x/tools/internal/lsp/signature"
	"golang.org/x/tools/internal/lsp/types"
)

type structy struct {
	x signature.MyType
}

func X(_ map[signatures.Alias]types.CoolAlias) (map[signatures.Alias]types.CoolAlias) {
	return nil
}

func _() {
	X() //@signature(")", "X(_ map[signatures.Alias]types.CoolAlias) map[signatures.Alias]types.CoolAlias", 0)
	_ = signature.MyType{} //@item(literalMyType, "signature.MyType{}", "", "var")
	s := structy{
		x: //@snippet(" //", literalMyType, "signature.MyType{\\}", "signature.MyType{\\}")
	}
}