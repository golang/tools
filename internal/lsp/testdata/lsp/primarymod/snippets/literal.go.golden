-- X(_ map[signatures.Alias]types.CoolAlias) map[signatures.Alias]types.CoolAlias-signature --
X(_ map[signatures.Alias]types.CoolAlias) map[signatures.Alias]types.CoolAlias

